src/main/scala/com/microsoft/hyperspace/actions/CreateAction.scala

```diff
@@ -72,4 +75,16 @@ class CreateAction(
   // TODO: The following should be protected, but RefreshAction is calling CreateAction.op().
   //   This needs to be refactored to mark this as protected.
   final override def op(): Unit = write(spark, df, indexConfig)
+
+  final override protected def event(appInfo: AppInfo, message: String): HyperspaceEvent = {
+    // LogEntry may or may not exist the first time an index is being created.
+    // This can happen, for example, if index config we receive is incompatible with the
+    // dataframe. So we use Try here. We still need an index object with empty values for event
+    // logging.
+    val index: Option[IndexLogEntry] = Try {
+      // If a logEntry exists, we can create a valid index object directly.
```

Comment:
this comment doesn't seem to be useful anymore?

Reply:
removed. Thanks @imback82